cycode/cli/printers/tables/table_printer_base.py

```diff
@@ -58,9 +61,12 @@ def _print_table(table: 'Table') -> None:
             click.echo(table.get_table().draw())
 
     @staticmethod
-    def _print_report_urls(local_scan_results: List['LocalScanResult']) -> None:
+    def _print_report_urls(local_scan_results: List['LocalScanResult'], aggregation_report_url: str = '') -> None:
         report_urls = [scan_result.report_url for scan_result in local_scan_results if scan_result.report_url]
-        if not report_urls:
+        if not report_urls and not aggregation_report_url:
+            return
+        if aggregation_report_url:
+            click.echo(f'Aggregation report URL: {aggregation_report_url}')
```

Comment:
let's not mark it as "aggregation" to the users. it's just report URL. the same as we had